src/collider/GeomTree.h

```diff
@@ -70,9 +70,9 @@ class GeomTree {
 	void RayTriIntersect(int numRays, const vector3f &origin, const vector3f *dirs, int triIdx, isect_t *isects) const;
 private:
 
-	int m_numVertices;
-	int m_numEdges;
-	int m_numTris;
+	size_t m_numVertices;
+	size_t m_numEdges;
+	size_t m_numTris;
 
```

Comment:
This should be `uint32_t` rather than `size_t` - Geom is not intended to operate on mesh data containing more than 4 billion vertices, triangles, or edges. Additionally, its upstream dependency, SingleBVHTreeBase, has a hard limit of < 2^32 triangles or edges.

Reply:
The m_num* members are now uint32_t.   If starsLimit is good as size_t then I'll squash the commits.

Reply:
size_t is appropriate for starsLimit - go ahead and squash.